test/test_helpers/consume_assessments_job_test_helper.rb

```diff
@@ -13,6 +13,14 @@ def self.response_statuses
     end
 
     def reported_symptom_assessment(symptomatic: nil)
+      if symptomatic
+        return {
+          response_status: nil,
+          threshold_condition_hash: @patient.jurisdiction.hierarchical_symptomatic_condition.threshold_condition_hash,
+          experiencing_symptoms: symptomatic,
+          patient_submission_token: @patient.submission_token
+        }.to_json
+      end
```

Comment:
Why was `experiencing_symptoms` removed from the JSON? I believe that experiencing symptoms is always present, just `nil` when not applicable.

Furthermore,

If symptomatic is provided as false this block of code will not execute, but should because it is simulating a "No" response to Experiencing symptoms. Should check for `nil?` instead.